data/fixtures/settings.yml

```diff
@@ -1351,6 +1351,11 @@ QubitSetting:
   Qubit_Settings_defaultArchivalDescriptionBrowseView:
     name: default_archival_description_browse_view
     value: table
+  Qubit_Settings_diacritics:
```

Comment:
You shouldn't need to add anything to this file. I think the setting will be dynamically created no matter what.

Reply:
if i remove these lines, i get a 500 error when i try to open up the diacritics settings page.

Reply:
Ah weird... I'll take a look.

Reply:
Can you try just changing line 33 to `"diacritics" => "0"`?

Reply:
(Hmm that doesn't seem to work. I'll poke around more.)

Reply:
Okay, this worked for me:

1. Change line 33 to `'diacritics' => '0'`
2. Change `DefaultEditAction::execute($request);` to `parent::execute($request);`
3. Remove the entire `processForm` function

Reply:
hmm were you able to upload a yaml file with those changes? i get the following error when uploading: `atom_1           | [18-Jul-2023 17:25:16] WARNING: [pool atom] child 15 said into stderr: "NOTICE: PHP message: This form is multipart, which means you need to supply a files array as the bind() method second argument."` and this error was the reason why I initially used `DefaultEditAction::execute($request);` as opposed to `parent::execute($request);`

Reply:
I was able to upload... I did run into that error but there was a change I made that resolved that. It's entirely possible there was a change I made that I forgot to add to the list, though, so I'll re-create the changes to see if I missed something!

Reply:
Yeah, I think I tweaked `apps/qubit/modules/settings/actions/editAction.class.php` to get it to work and forgot about it. Changing line 31 of that file to `$this->form->bind($request->getPostParameters(), $request->getFiles());` fixes things (basically just adding the second argument in the function call so the SettingsEditAction can work with multipart form data). So I'd say include that fix in your PR as well.

Reply:
ah right, i've tried that before but then it runs the parent functions and skips uploadDiaricits(), which is where all my error handling is, i'll play around more and see if there's another way for it to run the function! thanks mike!

Reply:
yay okay, i found a solution to that! lmk what you think of this solution!